vignettes/Status.Rmd

```diff
@@ -105,17 +105,17 @@ You'll notice that there are fewer rows in the `what_WQP` data frame (203) than
 
 Functioning normally, includes all available USGS data. The Water Quality Portal does not yet include new "WQX 3.0" profiles. 
 
-### Late February 2024
+### March 2024
 
-When the NWIS services are "frozen" (late February): no new USGS will be available via any Water Quality Portal query. 
+When the NWIS services are "frozen" (mid-March): no new USGS will be available via any Water Quality Portal query. 
 
 ### Fall 2024
 
 When the NWIS services are decommissioned (possible September 2024): Should not affect WQP functions.
 
-### March/April 2024
+### April 2024
 
-When WQP gets new "data profiles" (late March/April): New data profiles will include ALL USGS data (legacy and new). Shortly after these profiles are released, we plan to update dataRetrieval so that these new profiles are the default. 
+When WQP gets new "data profiles" (April 2024): New data profiles will include ALL USGS data (legacy and new). Shortly after these profiles are released, we plan to update dataRetrieval so that these new profiles are the default. 
```

Comment:
can you add something about availability of the v2 profiles? Something like: 'The WQX 2.0 profiles will continue to be available for an extended period of time.'

Reply:
This is the link I've used for a blog post: https://exchangenetwork.net/data-exchange/wqx/

Reply:
"The current WQP data profiles that  are considered "WQX version 2.0 profiles". These will remain available after the rollout of version 3.0. Eventually they will be retired, but there is not an estimated timeframe."

Does that sound OK? or is there a timeline?

Reply:
there is not an estimated timeframe, so this is accurate. In the past we've said something like: 'Eventually they will be retired: we will provide updated information as it becomes available.' But what you have is fine.